src/app/code/community/Omise/Gateway/Model/PaymentMethod.php

```diff
@@ -53,34 +47,73 @@ public function authorize(Varien_Object $payment, $amount)
      * @return Mage_Payment_Model_Abstract
      */
     public function capture(Varien_Object $payment, $amount)
-    {
+    {   
+
+        $order = $payment->getOrder();
         $authorized = isset($payment->getData('additional_information')['omise_charge_id']) ? $payment->getData('additional_information')['omise_charge_id'] : false;
         if ($authorized) {
             // Capture only.
             Mage::log('Start capture with OmiseCharge API!');
-
-            $charge = Mage::getModel('omise_gateway/omiseCharge')->captureOmiseCharge($authorized);
+            $charge = Mage::getModel('omise_gateway/omisecharge')->captureOmiseCharge($authorized);
```

Comment:
@duzydook please change this line `Mage::getModel('omise_gateway/omisecharge')` to `Mage::getModel('omise_gateway/omiseCharge')` krub (case-sensitive class name reason)